README.md

```diff
@@ -147,6 +148,10 @@ The only directive currently supported is `FormatBlockComment` and it's used as
 ;@AHK++FormatBlockCommentOff
 ```
 
+#### Formatter Known Issues
```

Comment:
Instead of adding a whole new `### Formatter Known Issues` section, let's take a cue from the debugger section and just add a note at the top of the `### Code Format` section:

```md
### Code Format

> There are some [known issues with the formatter](https://github.com/mark-wiemer/vscode-autohotkey-plus-plus/issues?q=is%3Aopen+is%3Aissue+label%3Aformatter), and we're always working to improve.

Supports standard VS Code formatting with a few options.
```

Reply:
Done.